src/Documentation/sidebar.json

```diff
@@ -24,14 +24,42 @@
       {
         "label": "Get Older Files",
         "slug": "older-versions"
+      }
+    ]
+  },
+  {
+    "slug": "tutorials",
+    "source": "tutorials/index.md",
+    "children": [
+      {
+        "slug": "basics",
```

Comment:
same here: it looks strange that we have `Interactive: Basics` out of `Interactive: Tutorials`. And that we have `Tutorials -> Interactive: Tutorials`. Can we move Basics into Interactive and keep only one page `interactive.md`?